src/index.js

```diff
@@ -2,113 +2,47 @@
-const span5 = generatorOfEl('span', null, null, '125', null, null);
-paragraph5.appendChild(span5);
-scoreItem5.appendChild(paragraph5);
-scoreList.appendChild(scoreItem5);
-
-const scoreItem6 = generatorOfEl('li', 'score-item', null, null, null, null);
-const paragraph6 = generatorOfEl('p', null, null, 'Name:', null, null);
-const span6 = generatorOfEl('span', null, null, '77', null, null);
-paragraph6.appendChild(span6);
-scoreItem6.appendChild(paragraph6);
-scoreList.appendChild(scoreItem6);
-
-const scoreItem7 = generatorOfEl('li', 'score-item', null, null, null, null);
-const paragraph7 = generatorOfEl('p', null, null, 'Name:', null, null);
-const span7 = generatorOfEl('span', null, null, '42', null, null);
-paragraph7.appendChild(span7);
-scoreItem7.appendChild(paragraph7);
-scoreList.appendChild(scoreItem7);
-
-mainDiv.appendChild(scoreList);
-main.appendChild(mainDiv);
-
-const form = generatorOfEl('form', 'add-scores', null, null, null, null);
-const formh2 = generatorOfEl('h2', null, null, 'Add your score', null, null);
-const firstDiv = generatorOfEl('div', 'name', null, null, null, null);
-const firstInput = generatorOfEl('input', 'name-input', 'text', null, 'Your name', 'name');
-firstDiv.appendChild(firstInput);
-form.appendChild(formh2);
-form.appendChild(firstDiv);
-const secondDiv = generatorOfEl('div', 'score', null, null, null, null);
-const secondInput = generatorOfEl('input', 'name-input', 'text', null, 'Your score', 'score');
-secondDiv.appendChild(secondInput);
-form.appendChild(secondDiv);
-const formbtn = generatorOfEl(
-  'button',
-  'submit-btn',
-  'submit',
-  'submit',
-  null,
-  null,
-);
-form.appendChild(formbtn);
-main.appendChild(form);
-
-const footer = generatorOfEl(
-  'footer',
-  'footer-section',
-  null,
-  null,
-  null,
-  null,
-);
-
-const wrapper = document.querySelector('.wrapper');
-wrapper.appendChild(header);
-wrapper.appendChild(main);
-wrapper.appendChild(footer);
\ No newline at end of file
+const xhr = new XMLHttpRequest();
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/6XXWCXn9VJxphtkrJMBV/scores/';
+
+const refresh = document.querySelector('.refresh');
+const domList = document.querySelector('.score-list');
+const nameValue = document.querySelector('.name-input');
+const scoreValue = document.querySelector('.score-input');
+const form = document.querySelector('.add-scores');
+
+const scores = new GetData();
+
+function displayEl(object) {
+  object.forEach((obj) => {
+    const li = generatorOfEl('li', 'score-item', null, null, null, null);
+    const paragraph = generatorOfEl('p', null, null, `${obj.user}:`, null, null);
+    const span = generatorOfEl('span', null, null, `${obj.score}`, null, null);
+    paragraph.appendChild(span);
+    li.appendChild(paragraph);
+    domList.appendChild(li);
+    if (domList.children.length > 0) {
+      domList.classList.add('border-on');
+    } else {
+      domList.classList.remove('border-on');
+    }
+  })
+}
```

Comment:
- [ ] I would invite you to use `arrow functions` instead of the `function` keyword. Arrow functions, introduced in ES6 and provides a concise way to write functions in JavaScript.

Reply:
done